backend/schemas/user.py

```
@@ -1,8 +1,62 @@
     username: str
     email: str
 
+# ✅ Signup Schema (Minimal Fields)
 class UserCreate(UserBase):
     password: str
+
+# ✅ Profile Completion Schema (Handled Separately)
+class ProfileCompletionRequest(BaseModel):
+    university_name: str
+    department: str
+    fields_of_interest: List[str]  # Expecting a list
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)